docs/application/dotnet/api/overview.md

```diff
@@ -24,7 +25,11 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
 
 A list of limitations is available at [Current Xamarin.Forms limitations](xamarin.forms-limitations.html).
 
-You can efficiently add Tizen wearable device specific user interfaces. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI/).
+## Tizen.Wearable.CircularUI
+
+[Tizen.Wearable.CircularUI](https://samsung.github.io/Tizen.CircularUI/index.html) provides Tizen wearable-specific user interfaces. Tizen.Wearable.CircularUI is an extension API of Xamarin.Forms. But these APIs only support Tizen wearable devices which support Tizen .NET, unlike Xamarin.Forms which supports cross platforms.
+
+You can efficiently add Tizen wearable-specific interfaces to your Tizen wearable application. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI).
```

Comment:
**Please ignore this comment if it is not in the scope of this review:**

This page refers to the APIs as: Tizen.Wearable.CircularUI 
The API reference page title is: Tizen CircularUI
The Github page intro title: Tizen Wearable CircularUI (repo name seems Tizen.CircularUI)
Should we consider to refer the API title conistently?

Also on the Github page: I happened to notice some typo:
Tizen Wearable CircularUI project is to develop an open source software motivate software developer to creating Tizen Wearable Xamarin Forms app more easily and efficiently.

Suggestion: (Not sure if I have understood the intention clearly, please verify)
Tizen Wearable CircularUI project is an open source software to motivate and help application developers to create Tizen Wearable Xamarin Forms apps more easily and efficiently.